src/Sylius/Behat/Context/Ui/Admin/ProductVariantsCreationContext.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Context\Ui\Admin;
+
+use Behat\Behat\Context\Context;
+use Sylius\Behat\Page\Admin\Crud\CreatePageInterface;
+use Sylius\Component\Core\Formatter\StringInflector;
+use Sylius\Component\Core\Model\ChannelInterface;
+use Sylius\Component\Core\Model\ProductInterface;
+
+final class ProductVariantsCreationContext implements Context
+{
+    public function __construct(private CreatePageInterface $createPage)
+    {
+    }
+
+    /**
+     * @When /^I create a new ("[^"]+") variant priced at "(?:€|£|\$)([^"]+)" for ("[^"]+" product) in the ("[^"]+" channel)$/
+     */
+    public function iCreateANewVariantPricedAtForProductInTheChannel(
```

Comment:
Why didn't you put this step in `ManagingProductVariantsContext`? IMO it's ok, just asking 😄

Reply:
I wanted to make it more reusable. With this approach for me, this context strictly means creating product variants and I think it’s more readable. That’s why I created a new context, I think I should change it also in API contexts. What do you think?